### PR TITLE
Style changes in RTR scripts

### PR DESCRIPTION
**Pull request by antoniomanuelfr**
## Description
Hi team! this PR aims to fix some style problems in RTR scripts.

## Tests
RTR pass on:

- [x] shared_modules/utils 
- [x] shared_modules/rsync
- [x] shared_modules/dbsync
- [x] data_provider
- [x] wazuh_modules/syscollector

### CODE REVIEW

**Review by Molter73 — APPROVED** [review]
LGTM!

File: src/ci/utils.py [hunk 1]
```diff
@@ -380,23 +473,28 @@ def runAStyleCheck(moduleName):
...
+    out = subprocess.run(astyleCommand, stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE, shell=True)
 
     if out.returncode == 0 and not out.stderr:
         stdoutString = str(out.stdout)
+
         if (stdoutString.find("Formatted") != -1):
             printFail('One or more files do not follow the Coding Style convention.')
-            printFail('Execute astyle --options=ci/input/astyle.config moduleName/*.h moduleName/*.cpp for further information.')
+            printFail('Execute astyle --options=ci/input/astyle.config moduleName/*.h moduleName/*.cpp for further'
```
Comment (konangc87) [inline src/ci/utils.py:484]: This seems to be a legacy issue but I believe it won't print the failing module correctly.

**Review by tinachoB — COMMENTED** [review]
The changes look good to me. @antoniomanuelfr , could you please start, manually, the CI checks so that we ensure everything is working as expected? Once this is done, I'll proceed with my approval. Good job ;)

**Review by konangc87 — DISMISSED** [review]

File: src/ci/utils.py [hunk 2]
```diff
@@ -142,111 +203,127 @@ def checkCoverage(output):
...
     if not os.path.exists(reportFolder):
         os.mkdir(reportFolder)
 
     for dir in glob.glob(moduleCMakeFiles):
         folders += '--directory ' + dir + ' '
-    includeDir = currentDir.replace("ci/../","")
-    coverageCommand = 'lcov ' + folders + ' --capture --output-file ' + reportFolder + '/code_coverage.info -rc lcov_branch_coverage=0 --exclude="*/tests/*" --include "'+includeDir+'/*" -q'
+
+    coverageCommand = f'lcov {folders} --capture --output-file {reportFolder}/code_coverage.info -rc ' \
+                      f'lcov_branch_coverage=0 --exclude="*/tests/*" --include "{includeDir}/*" -q'
```
Comment (jnasselle) [inline src/ci/utils.py:288]: PEP8 linter show "continuation line over-indented for visual indent" [E127](https://www.flake8rules.com/rules/E127.html)
```suggestion
    coverageCommand = f'lcov {folders} --capture --output-file {reportFolder}/code_coverage.info -rc ' \
                       f'lcov_branch_coverage=0 --exclude="*/tests/*" --include "{includeDir}/*" -q'
```

**Review by jnasselle — CHANGES_REQUESTED** [review]

File: src/ci/utils.py (hunk 2 above)
Comment (jnasselle) [reply src/ci/utils.py:288]: Hi agail @antoniomanuelfr ,

There are multiples python linters (flake8, pylint), and only in one of them his change was suggested. But this change generate alerts in the other ones, so i will mark this as solved.

Sorry about this misunderstanding. 

Regards,
Nico

**Review by jnasselle — APPROVED** [review]

**Review by tinachoB — APPROVED** [review]
LGTM!

**Pull request merged**